### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by dara214** [maxplayle04/HangoutBotPublic#5] [Ideas]
Instead of the command to flick through pages make it a reaction scroll through to make it easier; @maxplayle04

**Comment by iHelpMax** [comment]
**Woah!** Please don't directly tag the maintainers of this repository. They'll give a read when they see it! 👍

**Comment by thatmaxplayle** [comment]
We'll look into it, but it's really not something we're prioritising.
